swagger.json

```diff
@@ -527,7 +527,7 @@
                         "schema": {
                             "type": "object",
```

Comment:
No. see this change that I am reverting:
![regexample](https://cloud.githubusercontent.com/assets/23071967/23341206/b369a4a4-fc4b-11e6-941b-42c4a2359557.png)